CHANGELOG.md

```diff
@@ -6,6 +6,9 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 
 ## [Unreleased]
 ### Added
+- Hosts can authenticate from Google Compute Engines (GCE) using a Google ID token. See
```

Comment:
@orenbm I dont think the term Google ID token is the right term... maybe GCE instance identity token https://cloud.google.com/compute/docs/instances/verifying-instance-identity#request_signature

Reply:
thanks, updated.